test/common/src/main/kotlin/org/codice/compliance/verification/binding/BindingVerifier.kt

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compliance.verification.binding
+
+import com.google.api.client.http.HttpStatusCodes
+import org.codice.compliance.SAMLComplianceException
+import org.codice.compliance.SAMLSpecRefMessage
+
+open class BindingVerifier {
+    companion object {
+        private const val HTTP_ERROR_THRESHOLD = 400
+        /**
+         * Verifies the http status code of the response is not an error status code
+         * according to the binding spec
+         * 3.4.6 & 3.5.6 Error Reporting
+         */
+        fun verifyHttpStatusCode(code: Int) {
+            if (code >= HTTP_ERROR_THRESHOLD) {
+                throw SAMLComplianceException.createWithPropertyMessage(
+                        SAMLSpecRefMessage.SAMLBindings_3_5_6_a,
+                        property = "HTTP Status Code",
+                        actual = code.toString(),
+                        expected = "${HttpStatusCodes.STATUS_CODE_OK}"
```

Comment:
❓ Other codes are acceptable but we're printing out an expected value of 200. Is that okay, or should be be more broad in our expected value message?

Reply:
Whoops. Forgot to change that when Jeter and I decided to check for >400. Fixing now.